src/pages/productPage.ts

```diff
@@ -103,4 +111,14 @@ export default class ProductPage extends BasePage {
       await expect(this.productPageLocators.productCountInCompareListText).toHaveText(expectedRegex, { timeout: 15000 });
     });
   }
+
+  async verifyProductInWishlistByName(productName: string) {
```

Comment:
Use function names that clearly describe their purpose.
Maybe, "verifyAddToWishlistSuccessMessage" looks better? 
Because you don't check product name or image visability/price inside wishlish component.